### ISSUE TITLE
Confusion Matrix

### ISSUE THREAD

**Issue by trevorlapay** [trevorlapay/UNMFall18_ML_Project2_NB_LR#3]
I created a confusion matrix based on a split on the testing data and committed it to the repo. Interesting - index 7 newsgroup gets misclassified as every other type. I think one approach to getting better accuracy is to toss that class out so others don't get misclassified as that one. Running that now to see how we do.

**Comment by trevorlapay** [comment]
Update: it didn't do anything good

**Comment by hankyusa** [comment]
Perhaps we could make a binary classifier that learns to distinguish between that problem class and all others.

**Comment by trevorlapay** [comment]
Right, or figure out the problem words and eliminate them. The set minus
the bad newsgroup classified at 87%, which is still pretty good...
